shared/js/browser/macos-communication.js

```diff
@@ -92,6 +99,9 @@ const resolveInitialRender = function () {
     const isTrackerBlockingDataSet = typeof trackerBlockingData === 'object';
     const isLocaleSet = typeof locale === 'string';
     const isMaliciousSiteSet = maliciousSiteStatus && maliciousSiteStatus.kind !== undefined;
+
+    // wait for cert info
+    if (isInvalidCert === null) return console.log('isInvalidCert was not ready');
```

Comment:
here we are just refusing to continue until `isInvalidCert` is no longer null. Once it’s a true/false that’s our signal that the certificate data we have can be displayed